spec/models/post_spec.rb

```diff
@@ -1,5 +1,47 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'Post Model Validations' do
+    user = User.create(name: 'Dammy', bio: 'Dammy is so cool!')
+    subject do
+      Post.new(title: 'Bootstrap vs Tailwind', text: 'Let\'s  talk about Bootstrap and Tailwind', author_id: user)
+    end
```

Comment:
Please add a unit test for `#update_posts_counter` of Post model as it's required to test custom model methods.